src/fund/components/FundCardPaymentMethodDropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { background, border, cn } from '../../styles/theme';
+
+import { useTheme } from '../../core-react/internal/hooks/useTheme';
+import type {
+  FundCardPaymentMethodDropdownPropsReact,
+  PaymentMethodReact,
+} from '../types';
+import { FundCardPaymentMethodSelectRow } from './FundCardPaymentMethodSelectRow';
+import { FundCardPaymentMethodSelectorToggle } from './FundCardPaymentMethodSelectorToggle';
+import { useFundContext } from './FundCardProvider';
+
+export function FundCardPaymentMethodDropdown({
```

Comment:
Should be able to close on Esc and either close when user tabs out of the dropdown (or trap focus).

Sometimes I also get this flash

https://github.com/user-attachments/assets/5e3bb4a1-f542-4cff-a079-f23cd3b34746